src/app/loadout-builder/ClosableContainer.tsx

```diff
@@ -6,15 +6,17 @@ import styles from './ClosableContainer.m.scss';
  */
 export default function ClosableContainer({
   children,
+  enabled,
```

Comment:
You could default this to true.

Reply:
Done, I never normally use default props but I don't mind it here I suppose.